actions.c

```diff
@@ -2761,6 +2764,16 @@ cmd_exec(int interactive, struct cmdarg **args)
 	return cmdret_new(RET_SUCCESS, NULL);
 }
 
+cmdret *
+cmd_execw(int interactive, struct cmdarg **args)
+{
+	int status = -1;
+	pid_t pid = spawn(ARG_STRING(0), current_frame(rp_current_vscreen));
+	if (waitpid(pid, &status, 0) == -1) perror("cmd_execw");
+	else status = WEXITSTATUS(status);
```

Comment:
Please wrap/indent these properly

Reply:
Sorry, it looks right to me? I noticed elsewhere in the code single-statement conditional branches do not have bracing (which is my own preferred style also), but maybe you just want newlines like this?
```
        if (waitpid(pid, &status, 0) == -1)
                perror("cmd_execw");
        else
                status = WEXITSTATUS(status);
```
let me know if you want braces also or any other changes?

Reply:
Yes, just the newlines.  Braces can be omitted for single lines.